docs/how-to-build-with-ddn/overview.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 1
+sidebar_label: Basics
+description: "Learn how to get started with Hasura DDN and your GraphQL API."
+keywords:
+  - hasura ddn
+  - graphql api
+  - quickstart
+  - getting started
+  - guide
+---
+
+import { OverviewTopSectionIconNoVideo } from "@site/src/components/OverviewTopSectionIconNoVideo";
+import { OverviewPlainCard } from "@site/src/components/OverviewPlainCard";
+import Icon from "@site/static/icons/speedometer-04.svg";
+import Link from "@docusaurus/Link";
+
+# Basics
+
+## Introduction
+
+Hasura is a powerful-yet-simple-to-create API. Each tutorial in this section will guide you through creating your first
+Hasura DDN API on top of your preferred data source.
+
+### You have data
+
+Data is a broad term. What's our definition? It can be anything. No, really: anything.
+
+We realize that data lives in a variety of places—relational databases, non-relational databases, vector stores, OLAP,
+OLTP, and even generated on-the-fly by your functions...and you probably want to access all of it from a single
+endpoint.
+
+### You connect it to Hasura
+
+Regardless of where your data lives, you'll connect it to your API using a **native data connector**, or data connector
+for short. These connectors will introspect your data sources and generate metadata that Hasura DDN will use to generate
+your API.
+
+### You configure your API in metadata
+
+This **semantic metadata layer** is written as Hasura Metadata Language (HML), which is an extension of YAML. You can
+use the DDN CLI and your text editor to fine-tune and configure the API to your liking; this includes easily creating
+metadata objects for permissions, relationships, configuring authentication, etc.
```

Comment:
```suggestion
This **semantic metadata layer** is written in Hasura Metadata Language (HML), which is an extension of YAML. You can edit it with your text editor and the DDN CLI, configuring objects for auth, relationships, and all other aspects of the resultant API.
```

Also, not sure here if using the full-blown "semantic metadata layer" is adding value instead of just metadata.

Reply:
Thanks, @seanparkross  — I took the majority of your suggestions save this one. It's a quick simplicity nod to the concept of a metadata layer, so it should stay to introduce the concept to the user. Additionally, it's subtle, but listing the CLI first introduces it as the primary authoring tool...which, aside from hand-crafting certain metadata objects, it's what users will reach for when creating the majority of their API.